app/src/main/res/layout/row_saving_account_transaction.xml

```diff
@@ -35,15 +35,15 @@
             android:layout_height="wrap_content"
             android:layout_width="match_parent"
             android:text="@string/small_text"
-            android:textAppearance="@style/Base.TextAppearance.AppCompat.Medium"
+            android:textAppearance="@style/Base.TextAppearance.AppCompat.Small"
```

Comment:
Why did you change it to small as it can work well with Medium also?

Reply:
The text on xml represented it as small text and i found that it created inconsistency in size and it look irregular.